cruise-control/src/main/java/com/linkedin/kafka/cruisecontrol/servlet/parameters/PartitionLoadParameters.java

```diff
@@ -97,10 +97,14 @@ protected void initParameters() throws UnsupportedEncodingException {
     _endMs = ParameterUtils.endMs(_request);
     _partitionLowerBoundary = ParameterUtils.partitionBoundary(_request, false);
     _partitionUpperBoundary = ParameterUtils.partitionBoundary(_request, true);
-    _entries = ParameterUtils.entries(_request);
     _minValidPartitionRatio = ParameterUtils.minValidPartitionRatio(_request);
     _allowCapacityEstimation = ParameterUtils.allowCapacityEstimation(_request);
     _brokerIds = ParameterUtils.brokerIds(_request, true);
+    _entries = ParameterUtils.entries(_request);
```

Comment:
Looks like `ParameterUtils #entries(HttpServletRequest)` is used by other callers. Would it ever be acceptable to get a negative value for entries? If not, should we move the sanity check to `ParameterUtils #entries(HttpServletRequest)`?

Reply:
That's a good idea. All `entries` should have positive values. I will move the check to `ParameterUtils`. Thanks